CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v0.8.3, 16 January 2018
+
+Add support for `Onfido::SdkToken` resource
+
```

Comment:
There have actually been a large number of commits since 8.2.0. Do we need to include details of any of them in the changelog?

Reply:
They are mostly dependency version bumps so I won't include the details here.